ynr/apps/people/merging.py

```diff
@@ -311,6 +311,12 @@ def deep_merge_related_membership_objects(self, msource, mdest):
                 mdest.elected = msource.elected
                 mdest.save()
 
+        # copy any previous party allifiations if this is for a welsh run ballot
+        if msource.ballot.is_welsh_run:
```

Comment:
Have you missed out a commit with `is_welsh_run` in?

Reply:
Sorry, I see it's based off the other branch!

Reply:
Yep sorry bit confusing, I pushed that straight to the feature branch (along with the model/migration change)